### PR TITLE
Android Version Doc

### PR DESCRIPTION
**Pull request by davidbielik**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+nav_title: Android 11 Upgrade Guide
+page_order: 0
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:3]:
```suggestion
page_order: 10
```

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Links from push notifications will continue to work.
+
+### HTML In-App Messages
+
+Due to a change in **Android 11**'s WebView settings, HTML In-App Messages will not be displayed on **Android 11** Targeted apps until upgrading to [Braze Android SDK v8.1.0][1]. 
+ 
+For more information on other **Android 11** behavior changes unrelated to your Braze integration please review the official [Android 11 Behavior Changes](https://developer.android.com/preview/behavior-changes-11)
+
+## Android 11 Features
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:42]:
```suggestion
## Braze-related Android 11 Features
```

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+For a full migration guide to Android 11 features, see the [Android Developer Documentation](https://developer.android.com/preview/migration).
+
+## Upgrade Requirements
+
+If your Android app does not yet target the new **Android 11** (API Version 30), then no upgrade steps are required yet.
+
+When your Android app targets the new **Android 11** (API Version 30), please upgrade to the [Braze Android SDK v8.1.0][1].
+
+{% alert warning %}
+Apps targeting **Android 11** that do not upgrade to [Braze Android SDK v8.1.0][1] will experience issues with deep linking from Braze UI components, and will fail to render HTML In-App Messages.
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:21]:
```suggestion
Apps targeting **Android 11** that do not upgrade to Braze Android SDK v8.1.0 will experience issues with deep linking from Braze UI components and will fail to render HTML In-App Messages.
```

links in alerts don't render properly:

image

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+This guide describes relevant changes introduced in Android 11 (API version 30) and necessary upgrade steps required for the Braze Android SDK integration.
+
+For a full migration guide to Android 11 features, see the [Android Developer Documentation](https://developer.android.com/preview/migration).
+
+## Upgrade Requirements
+
+If your Android app does not yet target the new **Android 11** (API Version 30), then no upgrade steps are required yet.
+
+When your Android app targets the new **Android 11** (API Version 30), please upgrade to the [Braze Android SDK v8.1.0][1].
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:18]:
```suggestion
When your Android app targets **Android 11** (API Version 30), please upgrade to [Braze Android SDK v8.1.0][1].
```

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+# Android 11 SDK Upgrade Guide
+
+This guide describes relevant changes introduced in Android 11 (API version 30) and necessary upgrade steps required for the Braze Android SDK integration.
+
+For a full migration guide to Android 11 features, see the [Android Developer Documentation](https://developer.android.com/preview/migration).
+
+## Upgrade Requirements
+
+If your Android app does not yet target the new **Android 11** (API Version 30), then no upgrade steps are required yet.
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:16]:
```suggestion
If your Android app does not yet target **Android 11** (API Version 30), then no Braze Android SDK upgrade steps are required.
```

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+## Braze Android 11 Support
+
+Apps targeting **Android 11** (API 30) must upgrade to [Braze Android SDK v8.1.0][1].
+
+Failure to upgrade to [Braze Android SDK v8.1.0][1] for apps targeting **Android 11** will cause the following Braze features to no longer function as expected:
+ 
+### Deep Links
+
+Links to external URLs from a Braze message (In-App Message or Content Cards) will no longer work on **Android 11** Targeted apps until upgrading to [Braze Android SDK v8.1.0][1].
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:32]:
```suggestion
Deeplinks to external URLs from a Braze message (In-App Message or Content Cards) will no longer work on **Android 11** targeted apps until upgrading to [Braze Android SDK v8.1.0][1]. 
```

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+Apps targeting **Android 11** (API 30) must upgrade to [Braze Android SDK v8.1.0][1].
+
+Failure to upgrade to [Braze Android SDK v8.1.0][1] for apps targeting **Android 11** will cause the following Braze features to no longer function as expected:
+ 
+### Deep Links
+
+Links to external URLs from a Braze message (In-App Message or Content Cards) will no longer work on **Android 11** Targeted apps until upgrading to [Braze Android SDK v8.1.0][1]. 
+
+Links from push notifications will continue to work.
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:34]:
```suggestion
Deeplinks from push notifications will continue to work.
```

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+### HTML In-App Messages
+
+Due to a change in **Android 11**'s WebView settings, HTML In-App Messages will not be displayed on **Android 11** Targeted apps until upgrading to [Braze Android SDK v8.1.0][1].
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:38]:
```suggestion
Due to a change in **Android 11**'s WebView settings, HTML In-App Messages will not properly display on **Android 11** targeted apps until upgrading to [Braze Android SDK v8.1.0][1]. 
```

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+Links to external URLs from a Braze message (In-App Message or Content Cards) will no longer work on **Android 11** Targeted apps until upgrading to [Braze Android SDK v8.1.0][1]. 
+
+Links from push notifications will continue to work.
+
+### HTML In-App Messages
+
+Due to a change in **Android 11**'s WebView settings, HTML In-App Messages will not be displayed on **Android 11** Targeted apps until upgrading to [Braze Android SDK v8.1.0][1]. 
+ 
+For more information on other **Android 11** behavior changes unrelated to your Braze integration please review the official [Android 11 Behavior Changes](https://developer.android.com/preview/behavior-changes-11)
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:40]:
```suggestion
For more information on all **Android 11** behavior changes, please review the official [Android 11 Behavior Changes](https://developer.android.com/preview/behavior-changes-11).
```

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+## Braze-related Android 11 Features
+
+Android 11 introduced [several changes](https://developer.android.com/preview/privacy/location#change-details) to how location permissions are granted.
+ 
+### Allow Once Permissions
+Users can now grant permissions, such as Location Collection, on a temporary, one-time basis (see the [Android Docs](https://developer.android.com/preview/privacy/location#one-time-access) for more information). Once an app is closed, or in the background for long enough, that permission will be revoked automatically. The app would need to re-request this permission when needed in the future. Apps that already follow recommended patterns of asking for location likely already support one-time permissions.
+
+![Android Allow Once Permission][3]{: height="230px" }
+
+
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:51]: Remove extra newline

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+Due to a change in **Android 11**'s WebView settings, HTML In-App Messages will not be displayed on **Android 11** Targeted apps until upgrading to [Braze Android SDK v8.1.0][1]. 
+ 
+For more information on other **Android 11** behavior changes unrelated to your Braze integration please review the official [Android 11 Behavior Changes](https://developer.android.com/preview/behavior-changes-11)
+
+## Braze-related Android 11 Features
+
+Android 11 introduced [several changes](https://developer.android.com/preview/privacy/location#change-details) to how location permissions are granted.
+ 
+### Allow Once Permissions
+Users can now grant permissions, such as Location Collection, on a temporary, one-time basis (see the [Android Docs](https://developer.android.com/preview/privacy/location#one-time-access) for more information). Once an app is closed, or in the background for long enough, that permission will be revoked automatically. The app would need to re-request this permission when needed in the future. Apps that already follow recommended patterns of asking for location likely already support one-time permissions.
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:47]:
```suggestion
Users can now grant permissions, such as Location Collection, on a one-time basis (see the [Android Docs](https://developer.android.com/preview/privacy/location#one-time-access) for more information). Once an app is closed, or in the background for long enough, that permission will be revoked automatically. The app would need to re-request this permission when needed in the future. Apps that already follow the recommended flow for requesting permissions for location will already support one-time permissions.
```

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+
+![Android Allow Once Permission][3]{: height="230px" }
+
+ 
+### Background Location 
+Android 11 will require apps to first request the foreground location permission, and then after the app is backgrounded it may prompt the user again for Background Location permission. 
+Customers using Geofences should ensure their app follows Android’s recommendations on collecting Background Location permission. For more information, see the [Android Docs](https://developer.android.com/preview/privacy/location#background-location).
+
+![Android Background Location Permission][2]{: height="230px" }
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:56]: Just remove this picture. It's tiny and not really needed.

image

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md (hunk 10 above)
Comment (davidbielik) [reply _docs/_developer_guide/platform_integration_guides/android/android_11.md:51]: subjective

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 13]
```diff
@@ -0,0 +1,60 @@
...
+## Braze-related Android 11 Features
+
+Android 11 introduced [several changes](https://developer.android.com/preview/privacy/location#change-details) to how location permissions are granted.
+ 
+### Allow Once Permissions
+Users can now grant permissions, such as Location Collection, on a temporary, one-time basis (see the [Android Docs](https://developer.android.com/preview/privacy/location#one-time-access) for more information). Once an app is closed, or in the background for long enough, that permission will be revoked automatically. The app would need to re-request this permission when needed in the future. Apps that already follow recommended patterns of asking for location likely already support one-time permissions.
+
+![Android Allow Once Permission][3]{: height="230px" }
+
+
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:51]:
```suggestion
```

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+Users can now grant permissions, such as Location Collection, on a temporary, one-time basis (see the [Android Docs](https://developer.android.com/preview/privacy/location#one-time-access) for more information). Once an app is closed, or in the background for long enough, that permission will be revoked automatically. The app would need to re-request this permission when needed in the future. Apps that already follow recommended patterns of asking for location likely already support one-time permissions.
+
+![Android Allow Once Permission][3]{: height="230px" }
+
+ 
+### Background Location 
+Android 11 will require apps to first request the foreground location permission, and then after the app is backgrounded it may prompt the user again for Background Location permission. 
+Customers using Geofences should ensure their app follows Android’s recommendations on collecting Background Location permission. For more information, see the [Android Docs](https://developer.android.com/preview/privacy/location#background-location).
+
+![Android Background Location Permission][2]{: height="230px" }
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:56]:
```suggestion
```

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+
+[1]: https://github.com/Appboy/appboy-android-sdk/blob/master/CHANGELOG.md#810
+[2]: {% image_buster /assets/img/android/android-11-background-location-permission.svg %}
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:59]:
```suggestion
```

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md (hunk 10 above)
Comment (davidbielik) [reply _docs/_developer_guide/platform_integration_guides/android/android_11.md]: but, fine.

Comment (davidbielik) [reply _docs/_developer_guide/platform_integration_guides/android/android_11.md]: you and your newlines

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 16]
```diff
@@ -0,0 +1,57 @@
+---
+nav_title: Android 11 Upgrade Guide
+page_order: 10
+platform: Android
+hidden: true
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:5]:
```suggestion
```

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+nav_title: Android 11 Upgrade Guide
+page_order: 10
+platform: Android
+---
+
+# Android 11 SDK Upgrade Guide
+
+This guide describes relevant changes introduced in Android 11 (API version 30) and necessary upgrade steps required for the Braze Android SDK integration.
+
+For a full migration guide to Android 11 features, see the [Android Developer Documentation](https://developer.android.com/preview/migration).
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:11]: to Android 11 features -> to Android 11

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 18]
```diff
@@ -0,0 +1,56 @@
...
+
+## Upgrade Requirements
+
+If your Android app does not yet target **Android 11** (API Version 30), then no Braze Android SDK upgrade steps are required.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:15]: Are we bolding Android 11 everywhere? There are some inconsistencies

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 19]
```diff
@@ -0,0 +1,56 @@
...
+
+When your Android app targets **Android 11** (API Version 30), please upgrade to [Braze Android SDK v8.1.0][1].
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:17]: add " or above" (same comment everywhere else)

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 20]
```diff
@@ -0,0 +1,56 @@
...
+
+{% alert warning %}
+Apps targeting **Android 11** that do not upgrade to Braze Android SDK v8.1.0 will experience issues with deep linking from Braze UI components and will fail to render HTML In-App Messages.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:20]: Might want to add something to the effect of "Targeting Android while using a version of the Braze Android SDK less than v8.1.0 is unsupported."

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 21]
```diff
@@ -0,0 +1,56 @@
...
+
+Apps targeting **Android 11** (API 30) must upgrade to [Braze Android SDK v8.1.0][1].
+
+Failure to upgrade to [Braze Android SDK v8.1.0][1] for apps targeting **Android 11** will cause the following Braze features to no longer function as expected:
+ 
+### Deep Links
+
+Deeplinks to external URLs from a Braze message (In-App Message or Content Cards) will no longer work on **Android 11** targeted apps until upgrading to [Braze Android SDK v8.1.0][1]. 
+
+Deeplinks from push notifications will continue to work.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:33]: We can remove this line, it doesn't really add useful information and might confuse people

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 22]
```diff
@@ -0,0 +1,56 @@
...
+Due to a change in **Android 11**'s WebView settings, HTML In-App Messages will not properly display on **Android 11** targeted apps until upgrading to [Braze Android SDK v8.1.0][1]. 
+ 
+For more information on all **Android 11** behavior changes, please review the official [Android 11 Behavior Changes](https://developer.android.com/preview/behavior-changes-11).
+
+## Braze-related Android 11 Features
+
+Android 11 introduced [several changes](https://developer.android.com/preview/privacy/location#change-details) to how location permissions are granted.
+ 
+### Allow Once Permissions
+Users can now grant permissions, such as Location Collection, on a one-time basis (see the [Android Docs](https://developer.android.com/preview/privacy/location#one-time-access) for more information). Once an app is closed, or in the background for long enough, that permission will be revoked automatically. The app would need to re-request this permission when needed in the future. Apps that already follow the recommended flow for requesting permissions for location will already support one-time permissions.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:46]: space?

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 23]
```diff
@@ -0,0 +1,56 @@
...
+
+![Android Allow Once Permission][3]{: height="230px" }
+
+### Background Location 
+Android 11 will require apps to first request the foreground location permission, and then after the app is backgrounded it may prompt the user again for Background Location permission.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:51]: space?

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md [hunk 24]
```diff
@@ -0,0 +1,56 @@
...
+
+If your Android app does not yet target **Android 11** (API Version 30), then no Braze Android SDK upgrade steps are required.
+
+When your Android app targets **Android 11** (API Version 30), please upgrade to [Braze Android SDK v8.1.0][1].
+
+{% alert warning %}
+Apps targeting **Android 11** that do not upgrade to Braze Android SDK v8.1.0 will experience issues with deep linking from Braze UI components and will fail to render HTML In-App Messages.
+{% endalert %}
+
+## Braze Android 11 Support
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/android_11.md:23]: We might want to consolidate all action items for Android 11 somewhere. They appear to be:
1) Update to 8.1.0+
2) Ensure you're following best practices for location perms

File: _docs/_developer_guide/platform_integration_guides/android/android_11.md (hunk 22 above)
Comment (davidbielik) [reply _docs/_developer_guide/platform_integration_guides/android/android_11.md:46]: what space?

Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/android/android_11.md:46]: between the header and content

**Pull request merged**